docs/user/estimate_migration.rst

```diff
@@ -0,0 +1,160 @@
+====================
+Migration Estimation
+====================
+
+Quick Start Guide
+-----------------
+
+1. Add credentials for source and destination cloud to `discover.yaml` file,
+   for example::
+
+    clouds:
+      src_grizzly:
+        credential:
+          auth_url: https://keystone.example.com/v2.0/
+          username: admin
+          password: admin
+          region_name: grizzly
+        scope:
+          project_id: 00000000000000000000000000000001
+        ssh:
+          username: foobar
+          sudo_password: foobar
+          connection_attempts: 3
+      dst_liberty:
+        credential:
+          auth_url: https://keystone.example.com/v2.0/
+          username: admin
+          password: admin
+          region_name: liberty
+        scope:
+          project_id: 00000000000000000000000000000002
+        ssh:
+          username: foobar
+          sudo_password: foobar
+          connection_attempts: 3
+
+2. Add information about migrations to `discover.yaml` file, for example::
+
+    migrations:
+      grizzly_to_liberty:
+        source: src_grizzly
+        destination: dst_liberty
+        objects:
+          vms:
+            - tenant.name: demo
+          images:
+            - tenant.name: demo
+          volumes:
+            - tenant.name: demo
+
+3. Run `fab estimate_migration:/path/to/discover.yaml,grizzly_to_liberty`.
+   Example::
+
+    $ fab estimate_migration:/path/to/discover.yaml,grizzly_juno_vms_images
+    Migration grizzly_juno_vms_images estimates:
+    Images:
+      Size: 842.3MB
+    Ephemeral disks:
+      Size: 121.3GB
+    Volumes:
+      Size: 0.0GB
+
+    10 largest servers:
+      1. 4fcc44be-583f-4f58-8f07-5fe703afaef8 poldb02 - 24.0GB
+      2. 4b1593ef-d8c3-414c-9ca8-2a48157b0fb7 pol01 - 16.6GB
+      3. 8f6f3beb-d4c5-434b-a96e-f92a6da22f3b syslog01 - 14.0GB
+      4. 64546943-30c7-402d-a42d-64c94d27b9d2 nagios-sat-red01 - 9.8GB
+      5. a1efe771-ecef-485d-97fa-843fc2578019 dispatch01 - 7.0GB
+      6. 324d79b3-551e-49ea-85a4-e8121c639dbb marketplace-ui01 - 6.1GB
+      7. e4ec887f-1c0e-4918-bb2e-62b54b3b953d authz01 - 6.0GB
+      8. 0fbac35b-7195-49d9-976e-23eb6b0a0d0b dispatchlb01 - 5.9GB
+      9. e2beafbb-670a-450a-9677-c2d2dc779888 support-ui01 - 5.8GB
+      10. a677dc39-7339-4a91-961b-11e5a3956b5e policy-ui01 - 5.7GB
+
+    Done.
+
+   On first run this command will download information about all objects in
+   cloud, then ssh into every active compute node that have VM instance
```

Comment:
Please specify whether information is downloaded from source, destination or both clouds, it's not clear from the context. Would also be good to add warning saying that this step may take long time for large clouds.